go-controller/pkg/ovn/policy_test.go

```diff
@@ -2229,6 +2232,66 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Low-Level Operations", func() {
 		gomega.Expect(gp.delNamespaceAddressSet(four.GetObjectID(libovsdbops.ObjectNameKey))).To(gomega.BeFalse())
 	})
 
+	ginkgo.It("creates stateless OVN ACLs based off of the annotation", func() {
```

Comment:
why is it a "Low-Level Operation" and not in "OVN NetworkPolicy Operations during execution" context?

Reply:
@npinaeva still it should be under low level network policy operations right? as we need to test if are we seeing a stateless ACL if we have the stateless annotation on the network policy. As 
`Tests AddAllowACLFromNode
computes match strings from address sets correctly`
as the above tests are under  low level operations which are dealing with underlying ACL changes for network policy so moved the stateless ACL cration under it

Reply:
I mean you can have low-level test is you want to check e.g. what `buildLocalPodACLs` returns for stateless gress, but I would expect to see "OVN NetworkPolicy Operations during execution" test too, that would create network policy with that annotation via k8s api and check the db for expected acls. 
I assume since the logic is not that complex, low-level test may not be needed if we have a higher-level version of it